features/cleaner/cleaner_chunks.go

```diff
@@ -0,0 +1,149 @@
+package cleaner
+
+import (
+	"app/tools"
+	processing "app/tools/processing"
+
+	"math"
```

Comment:
Looks like the auto-formatter wasn't triggered here. Please see if you can get this to format correctly.